src/configure/configure.ts

```diff
@@ -185,12 +186,26 @@ class Orchestrator {
             if (!this.inputs.azureSession) {
                 this.inputs.azureSession = getAzureSession();
             }
+
+            // Right click scenario not supported for Azure repo
+            if (this.isResourceAlreadySelected()) {
+                if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.AzureRepos) {
+                    throw Error(Messages.GithubRepoRequired);
+                } else if (!extensionVariables.enableGitHubWorkflow) {
+                    // For github repo, we create a github pipeline
+                    extensionVariables.enableGitHubWorkflow = true;
+                    vscode.window.showInformationMessage("Creating github pipeline.");
+                }
+            }
+
             const repoAnalysisResult = await this.getRepositoryAnalysis();
-            this.setPipelineType();
-            // Right click on not resource not supported for Azure pipeline
-            if (this.isResourceAlreadySelected() && this.pipelineType === PipelineType.AzurePipeline) {
-                throw Error("Scenario not supported for Azure pipelines.");
+
+            // Right click on not resource not supported for local repo
+            if (this.isResourceAlreadySelected() && !repoAnalysisResult) {
```

Comment:
`!repoAnalysisResult` will also be false when it is a valid github remote but repo analysis was not able to detect anything (as it might not have support). We should not block such case and allow use to to proceed with a default Dockerfile path.

Reply:
May be we can set a flag to indicate local repo in `await this.getSourceRepositoryDetails();` and use that flag here?

Reply:
yeah, that can be done.